application/templates/cms/_measures_with_version_history.html

```diff
@@ -2,7 +2,7 @@
 
   {% from "cms/measure_actions.html" import render_actions with context %}
 
-    <div class="reordering-save-status" class="" style="margin-top: 30px; font-size: 16px;">&nbsp;</div>
+    <div class="reordering-save-status" class="" style="font-size: 16px;">&nbsp;</div>
```

Comment:
I'm not super sure what this does, but IMO the design looks even better with it as `display: none;`?

There still seems to be quite a large gap between the subtopic name and the table with this as it is now.

Reply:
Hmm, didn't notice that when I was using it...

Reply:
So it does. Feels like that'd be nice to have inline rather than dedicate it a block of space that will be empty most of the time. Up to you if you can think of an easy way to address that now. If not, leave it as-is.

Reply:
@samuelhwilliams heh, maybe. I couldn't think of a good place to show it inline (eg do you show it next to the position moved from, or to? And what if you re-order a bunch in quick succession?). I did consider hiding it when not in use, but then the table jumps down to show you the status, which is jarring.

Could probably still take up a bit less space though.

Reply:
Yeah, jumping would be super awful. Let's not do that.